### PR TITLE
docs: add WSL setup instructions

### PR DESCRIPTION
**Pull request by geoffrich**
# Alaska Airlines Pull Request

This adds setup instructions for WSL.

Reference: https://github.com/AlaskaAirlines/WC-Generator/issues/89

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] New capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: docs/devSetup.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+Inside VS Code, open the integrated terminal with CTRL+SHIFT+`. Run the following commands to install dependencies and build the component.
+
+```sh
+npm i
+npm run dev
+```
+
+In a separate terminal, run `npm run serve` to serve the component's demo page. CTRL+click on the displayed URL to open the demo page in your web browser. You should see the auro-button demo page.
```
Comment (blackfalcon) [inline docs/devSetup.md:66]:
```suggestion
In a separate terminal/shell, run `npm run serve` to serve the component's demo page. `CTRL+click` on the displayed URL to open the demo page in your web browser. You should see the `auro-button` demo page.
```

File: docs/devSetup.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Finally, run the component tests with `npm run test` and make sure everything passes.
+
+### Set up git
+
+You need to configure your git name and email in WSL. Run the following in the terminal:
```
Comment (blackfalcon) [inline docs/devSetup.md:72]:
```suggestion
You need to configure your Git name and email in WSL. Run the following in the terminal:
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: docs/devSetup.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+## Windows
+
+If you are contributing to Auro on Windows, [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about) is a must. Otherwise, you will encounter issues with the Auro repos. Here are the steps for setting up your development environment to work on Auro.
+
+### Installing WSL
+
+Install WSL using [Microsoft's guide](https://docs.microsoft.com/en-us/windows/wsl/install-win10). Note the [version requirements](https://docs.microsoft.com/en-us/windows/wsl/install-win10#step-2---check-requirements-for-running-wsl-2) before starting. You may have to update Windows first. If you run into issues, check their [troubleshooting docs](https://docs.microsoft.com/en-us/windows/wsl/install-win10#troubleshooting-installation). If you don't know which Linux distro to pick, choose Ubuntu.
+
+After WSL is installed, you can open the WSL terminal by running the `wsl` command or opening Ubuntu from the start menu. WSL does not have a GUI you can interact with -- everything is done from the command line. You may want to find a cheat sheet if you are unfamiliar with Bash.
```
[the remark this answers is not in the document]
Comment (geoffrich) [reply docs/devSetup.md:11]: I was having trouble finding a high-quality one so I skipped it. Looks like the one you linked is more for bash scripting, so not super relevant to moving around the terminal. What do you think about [this one](https://www.git-tower.com/blog/command-line-cheat-sheet/)?

File: docs/devSetup.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+### Installing dependencies
+
+You need to install several dependencies on WSL so that you can develop Auro. Even if you have these installed on your main Windows machine, you still need to install them in WSL.
```
[the remark this answers is not in the document]
Comment (geoffrich) [reply docs/devSetup.md:15]: I'm trying to draw a distinction here between developing with Auro (i.e. consuming the components in a React/Svelte/whatever project) and developing Auro components themselves. It's possible to consume the components without WSL. Do you think "with Auro" is clear enough?

File: docs/devSetup.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+```sh
+cd ~
+mkdir git
+cd git
+git clone https://github.com/AlaskaAirlines/auro-button.git
+cd auro-button
+code .
+```
+
+Inside VS Code, open the integrated terminal with CTRL+SHIFT+`. Run the following commands to install dependencies and build the component.
```
[the remark this answers is not in the document]
Comment (geoffrich) [reply docs/devSetup.md]: Fixed. Had to use double-backticks on the shortcut, since it's actually referring to the backtick key.

**Comment by geoffrich** [comment]
A couple Qs on your suggestions, but I committed the ones I agreed with. Let me know if you want me to squash everything into a single commit when we wrap up your other comments.

File: docs/devSetup.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (blackfalcon) [reply docs/devSetup.md:15]: I think so. 

> ... develop with Auro.

Maybe be more direct then

> ... develop an Auro component.

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**